app/assets/javascripts/discourse/app/components/modal/create-account.hbs

```diff
@@ -31,244 +31,243 @@
       </div>
 
       {{#if this.showCreateForm}}
-        <div class="login-form">
-          <form>
-            {{#if this.associateHtml}}
-              <div class="input-group create-account-associate-link">
-                <span>{{html-safe this.associateHtml}}</span>
-              </div>
-            {{/if}}
-            <div class="input-group create-account-email">
-              <Input
-                {{on "focusout" (action "checkEmailAvailability")}}
-                @type="email"
-                @value={{this.model.accountEmail}}
-                disabled={{this.emailDisabled}}
-                autofocus="autofocus"
-                aria-describedby="account-email-validation"
-                aria-invalid={{this.emailValidation.failed}}
-                name="email"
-                id="new-account-email"
-                class={{value-entered this.model.accountEmail}}
+        <form id="login-form">
```

Comment:
@chapoi is there a particular reason for switching from `class="login-form"` to `id="login-form"`? If not, perhaps we should stick with `class=` to reduce the potential impact on themes/plugins?

Reply:
I didn't switch, they were both in use and out of safety I just kept both